src/Tests/Microsoft.NET.Build.Containers.IntegrationTests/CreateNewImageTests.cs

```diff
@@ -31,7 +31,7 @@ public void CreateNewImage_Baseline()
 
         newProjectDir.Create();
 
-        new DotnetNewCommand(_testOutput, "console", "-f", ToolsetInfo.CurrentTargetFramework)
+        new DotnetNewCommand(_testOutput, "console", "-f", ToolsetInfo.NextTargetFramework)
```

Comment:
Do we have an issue filed to track swapping 9.0 to current target framework (probably in Dec or Jan after all the templates have moved forward and 9 has stabilized some)?